### PR TITLE
ddl: Make verification background job more stable

### PR DESCRIPTION
**Pull request by zimulala**
Make verification background job more stable and add a test for dropping the database.

### CODE REVIEW

File: ddl/bg_worker.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func (d *ddl) handleBgJobQueue() error {
 
 // runBgJob runs a background job.
 func (d *ddl) runBgJob(t *meta.Meta, job *model.Job) {
+	log.Infof("[ddl] run background job %s", job)
```
Comment (shenli) [inline ddl/bg_worker.go]: Can we use debug level?

File: ddl/bg_worker_test.go [hunk 2]
```diff
@@ -42,20 +42,31 @@ func (s *testDDLSuite) TestDropSchemaError(c *C) {
...
-		c.Assert(err, IsNil)
-		c.Assert(historyBgJob, NotNil)
-		c.Assert(historyBgJob.State, Equals, state)
-
-		return nil
-	})
+func verifyBgJobState(c *C, d *ddl, job *model.Job, state model.JobState, sleepTime time.Duration) {
+	time.Sleep(sleepTime)
+	var isDone bool
+	for i := 0; i < 50; i++ {
```
Comment (shenli) [inline ddl/bg_worker_test.go]: Will this cost too long time for unit test？

Comment (coocood) [inline ddl/bg_worker_test.go]: Most of the time, it breaks at the first loop.

**Comment by coocood** [comment]
LGTM

File: ddl/bg_worker_test.go (hunk 2 above)
Comment (shenli) [inline ddl/bg_worker_test.go]: ok

File: ddl/bg_worker.go (hunk 1 above)
Comment (zimulala) [inline ddl/bg_worker.go]: `runDDLJob` uses info level, so here I use info level.

**Comment by shenli** [comment]
LGTM

**Pull request merged**